assets/javascripts/modules/autoComplete.js

```diff
@@ -351,6 +351,17 @@ var suggestionsEvent = function () {
 
 
 /**
+  * If a select elem is being used, load value from there
+**/
+var loadExistingValue = function () {
```

Comment:
you could send in the `$autoCompleteInputElem` to make this not rely on a global and become more functional